src/Microsoft.ML.Data/Prediction/IPredictionTransformer.cs

```diff
@@ -2,21 +2,18 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using Microsoft.ML.Core.Data;
 using Microsoft.ML.Runtime.Data;
-using Microsoft.ML.Runtime.Internal.Calibration;
-using System;
-using System.Collections.Generic;
-using System.Text;
 
 namespace Microsoft.ML.Runtime
 {
     public interface IPredictionTransformer<out TModel> : ITransformer
```

Comment:
>IPredictionTransformer [](start = 21, length = 22)

I don't think we even need an interface for FFM, since for the time being it's the only trainer that accepts multiple feature columns.

Reply:
If we want to inquire about all trainers, it is useful to have them extend one interface.